pages/molecules/gallery-image/gallery-image.tsx

```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
-import { GalleryImage } from "../../../data";
+import { GalleryImageData } from "../../../data";
```

Comment:
you could also use import alias instead of renaming the interface
e.g. 
import { GalleryImage as GallerImageData } from '../../../data';

resolve when read